sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_generated/aio/operations/_operations.py

```diff
@@ -193,10 +84,11 @@ async def get_by_id(self, id: str, **kwargs: Any) -> AsyncIterator[bytes]:
         path_format_arguments = {
             "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
         }
-        request.url = self._client.format_url(request.url, **path_format_arguments)  # type: ignore
+        request.url = self._client.format_url(request.url, **path_format_arguments)
 
-        pipeline_response = await self._client._pipeline.run(  # type: ignore # pylint: disable=protected-access
-            request, stream=True, **kwargs
+        _stream = True
+        pipeline_response: PipelineResponse = await self._client._pipeline.run(  # pylint: disable=protected-access
+            request, stream=_stream, **kwargs
         )
 
         response = pipeline_response.http_response
```

Comment:
how would this response be typed?
currently this returns a `<class 'azure.core.rest._requests_basic.RestRequestsTransportResponse'>` object. But typing it as azure.core.pipeline.transport.HttpResponse results in an error when calling `response.read()`, since `read()` is not an attribute in HttpResponse.

Reply:
import HttpResponse from azure.core.rest, not azure.core.pipeline.transport